client/utils/insertActions.js

```diff
@@ -0,0 +1,5 @@
+export default {
```

Comment:
`activityInsertLocation` seems like best option for me but I'm not sure about not destructuring since in that case we would end up with huge if statement e.g. `action === activityInsertLocation.BEFORE`. Also, with this rename it would make sense to rename `action` prop to `location` or `activityLocation`?

Reply:
Enum likes should be PascalCase in the codebase ;) That's fine, you can rename it upon import within context to InsertLocation or similar ;) I think when something is extracted to a bit more general section it should have a bit more detailed definition. I think action is still better because of reorder? Thoughts?